dvc/progress.py

```diff
@@ -35,6 +35,7 @@ class Tqdm(tqdm):
     BYTES_DEFAULTS = dict(
         unit="B", unit_scale=True, unit_divisor=1024, miniters=1
     )
+    MAX_BARS = 20  # maximum parallel bars to display
```

Comment:
ideally auto-detect; see https://github.com/tqdm/tqdm/blob/master/tqdm/utils.py#L263-L330 for inspiration